.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,53 +1,8 @@
 ## Pull Request checklist
 
-Please check if your PR fulfills the following requirements:
+Please review the below PR requirements:
 
-- [ ] Tests for the changes have been added (for bug fixes / features)
-- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
 - [ ] Build was run locally and any changes were pushed
```

Comment:
I don't think this 'Build was run' is required because vercel already builds the PR so it can be verified there directly.

Reply:
I disagree. Opening a PR that doesn't build is sloppy and adds unnecessary noise.